docs/release-notes.md

```diff
@@ -1,47 +1,45 @@
 # Release Notes
 
 If you haven't had a chance - checkout the [Letter From Devs](./letter.md) for a message from our development team.
-## Key Feature Changes for 1.1
-This section outlines key features changes between versions 1.0 and 1.1 for [Open Active Management Technology (Open AMT) Cloud Toolkit.](Glossary.md#o) 
+## Key Feature Changes for 1.2
+This section outlines key features changes between versions 1.1 and 1.2 for [Open Active Management Technology (Open AMT) Cloud Toolkit.](Glossary.md#o) 
 
 ### Additions
 #### [RPS](Glossary.md#r)
-- **Network Configuration on Intel&reg; AMT devices:** Added feature to AMT Profiles to select configuring Intel&reg; AMT devices to use DHCP or Static IP.  When set for static IP, Intel&reg; AMT will sync with the Host OS static IP address.  Static IP Address is not supported with CIRA profiles.
-- **MEBx Password configuration:** The AMT Profile now supports configuration options for setting either a defined or randomized MEBx password on Intel&reg; AMT devices.
-- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
+- **Hostname and Device Metadata tags:** We have added the ability to define metadata tags for a each Intel&reg; AMT profile.  These tags will be applied to each device that is configured using the profile.  Additionally, if RPC sends up the Intel&reg; AMT device OS hostname during activation, RPS save this information as part of the device information.  When RPS configures a device, it will perform a POST call to MPS to register that device in MPS.  This works in conjunction with the improved metadata gathering that RPC added in 1.1.
```

Comment:
'during activation, RPS **will** save this information...'

Reply:
updated in amended commit